CONTRIBUTING.md

```diff
@@ -435,6 +436,13 @@ ghstack submit
 [`ghstack`](https://github.com/ezyang/ghstack). It creates a large commit that is
 of very low signal to reviewers.
 
+## Merging your Change
+If you know the right people or team that should approve your PR, add them to the Reviewers list.
+
+If not, your PR will get triaged and assigend to the appropriate person in a couple business days.
```

Comment:
```suggestion
If not, your PR will get triaged and assigned to the appropriate person in a couple business days. If this is not the case, please ping @pytorch/pytorch-dev-infra or bring your PR to our office hours.
```
And then link to OH. I feel like this is not as helpful as it can be if we don't account for the edge cases. (Otherwise, everything goes smoothly and people wouldn't need this documentation)

Reply:
I like the idea of sharing a backup plan.  I'd rather point them to the people who're actually responsible for triaging the PR instead of dev-infra though.  Let's leave dev infra office hours as the ultimate fallback